app/src/main/java/com/pokerio/app/screens/GameScreen.kt

```diff
@@ -49,7 +57,13 @@ fun GameScreen() {
     val orientation = ActivityInfo.SCREEN_ORIENTATION_USER_LANDSCAPE
     val systemUiController = rememberSystemUiController()
     var raiseDialogOpen by remember { mutableStateOf(false) }
-    var debugNumberOfActions by remember { mutableStateOf(0) }
+    var actions by remember { mutableStateOf(0) }
+    var players by remember(key1 = actions) {
+        mutableStateOf(GameState.players.toList())
+    }
+    var winningsPool by remember {
```

Comment:
I think this should also be marked with the `key` attribute - sure, the entire component will update anyway, but we might change some logic in the future and we don't want this to break.

Reply:
Done.